src/clientdlg.cpp

```diff
@@ -179,7 +179,35 @@ CClientDlg::CClientDlg ( CClient*         pNCliP,
     lblBuffers->setWhatsThis ( strLEDBuffers );
     ledBuffers->setWhatsThis ( strLEDBuffers );
 
-    ledBuffers->setAccessibleName ( tr ( "Buffers status LED indicator" ) );
+    ledBuffers->setAccessibleName ( tr ( "Jitter buffer status LED indicator" ) );
+
+    // current connection status parameter
+    QString strConnStats = "<b>" +
+                           tr ( "Current Connection Status "
+                                "Parameter" ) +
+                           ":</b> " +
+                           tr ( "The Ping Time is the time required for the audio "
+                                "stream to travel from the client to the server and back again. This "
+                                "delay is introduced by the network and should be about "
+                                "20-30 ms. If this delay is higher than about 50 ms, your distance to "
+                                "the server is too large or your internet connection is not "
+                                "sufficient." ) +
+                           "<br>" +
+                           tr ( "Overall Delay is calculated from the current Ping Time and the "
+                                "delay introduced by the current buffer settings." ) ;
+
+    lblPing->setWhatsThis ( strConnStats );
+    lblPingVal->setWhatsThis ( strConnStats );
+    lblDelay->setWhatsThis ( strConnStats );
+    lblDelayVal->setWhatsThis ( strConnStats );
+    ledDelay->setWhatsThis ( strConnStats );
+    ledDelay->setToolTip ( tr ( "If this LED indicator turns red, "
```

Comment:
@dcorson-ticino-com would it make sense to update the Fancy LEDs as part of this change? In total 6 lines of codes you need to update and add two images (which you can take from my PR #1752)?
As all other items in that PR have been covered now by this update.

Reply:
> Not part of this PR, but we should use QString ( … %1 … ) .arg ( APP_NAME ); here

Yes, I agree. Just to re-iterate, this not only does not have to be part of this PR, it *should* not be part of this PR to avoid introducing even more translation movements.